### DISCUSSION TITLE
Is this project work on windows

### DISCUSSION THREAD

**Discussion by Zabihullah-Danish** [mehedijaman/laravel-zkteco#3] [Q&A] [answered]
I have created a laravel project in windows and installed laravel-zkteco package for ZKT device integration but it is not working.
just the $zk->connect() return 1 and when run other functions not work at all below is the error when running other methos
unpack(): Type H: not enough input, need 1, have

**Comment by mehedijaman** [comment]
Are you using xampp on windows ? You didn't specify details about your configuration. 
Note that, you have to enable socket extension. Please refer to the package doc on how to enable socket extension.
And also this package works on device in the same network. I haven't tested it for devices from different network.

**Reply by Zabihullah-Danish** [reply]
Yes I am using xampp on windows and also enabled sockets extension and my device is also in the same network
$zk->connect() // return 1
all other methods give this error
![image](https://github.com/user-attachments/assets/fa888f9e-2f5a-4d38-ad87-a1c90e06d637)

**Reply by mehedijaman** [reply, marked as answer]
May be your device is not supporting. I have tested only on F18 and iClock9000 Model.
